pkg/common/utils.go

```diff
@@ -713,6 +713,13 @@ func ValidateDataCacheMode(s string) error {
 	return fmt.Errorf("invalid data-cache-mode %s. Only \"writeback\" and \"writethrough\" is a valid input", s)
 }
 
+func ValidateDataCacheSize(size int64) error {
```

Comment:
This can be a more generic function like ValidateNonNegativeInt, it can be used in a few other places. "Input should be set to > 0, got %d".

Reply:
And unit tests.